monero-sys/src/lib.rs

```diff
@@ -52,11 +88,14 @@ pub struct Wallet {
     wallet: FfiWallet,
     manager: WalletManager,
     call_receiver: UnboundedReceiver<Call>,
+    pending_transactions: Arc<Mutex<HashMap<Uuid, PendingTransaction>>>,
```

Comment:
This hashmap stays in the wallet thread and is only ever accessed by a single call at a time (since we don't use any async everything is blocking) -> This can just be `HashMap<Uuid, PendingTransaction>`, not need for Arc or Mutex